smartcontractkit/external-adapters-js
# Add support for Starkware to layer2 sequencer health EA

## Closes 46377

## Description

Add support to `layer2-sequencer-health` EA to track Starkware Sequencer health

## Changes

- Update Layer2 Sequencer Health EA to check for the Starkware Sequencer's health.
- This is a two step process
   1. Poll Sequencer directly for the pending block to see if either a new block is produced or if there are new pending transactions
   2. If step 1 fails, then try send a dummy transaction to the network and see if we get back an expected error.  Receiving an expected error means that the Sequencer is still healthy.

## Steps to Test

 - Run unit and integration tests
 - Run EA
 - Send request payload below and verify that the EA returns a successful response.  Repeat this with other networks.
 ```
{
    "data": {
        "network": "starkware"
    }
}
```

## Quality Assurance
